Library/Homebrew/test/tap_spec.rb

```diff
@@ -140,10 +140,6 @@ def setup_completion(link:)
     end
   end
 
-  specify "::names" do
-    expect(described_class.names.sort).to eq(["homebrew/core", "homebrew/foo"])
-  end
-
```

Comment:
This will be deprecated soonish so I figured we might as well remove the test instead of updating it. I guess we weren't automatically including the core cask tap on macOS before in this list.